### ISSUE TITLE
[BUG] Storybook with tailwing jit is broken on latest @nuxtjs/storybook

### ISSUE THREAD

**Issue by mathieucollet** [nuxt-community/storybook#254]
### Version
@nuxtjs/storybook: `4.0.0`
nuxt: `2.15.4`
@nuxtjs/tailwindcss: `4.0.1` or `4.0.2` or `4.0.3`

### Reproduction Link
[https://github.com/mathieucollet/storybooktest](https://github.com/mathieucollet/storybooktest)

### Steps to reproduce
1. Create a Nuxt project `yarn create nuxt-app whatever` with tailwind module
2. cd IN
3. Install storybook : `yarn add -D @nuxtjs/storybook postcss@latest`
4. Add the jit option to tailwind in `nuxt.config.js`:
```js
tailwindcss: {
  jit: true,
},
```
5. Launch Storybook: `nuxt storybook`

### What is Expected?
Storybook should be working.

### What is actually happening?
```bash
ERROR  Failed to compile with 1 errors
ERROR  Error 
No files matching '../.nuxt-storybook/storybook/storybook-init-framework-entry.js' were found.

...

ERROR in No files matching '../.nuxt-storybook/storybook/storybook-init-framework-entry.js' were found.
```

### Workaround
Removing jit from tailwind config solve the problem

### Anything else
Before the last update of `@nuxtjs/storybook` it was working fine even with jit option setted

**Comment by farnabaz** [comment]
Do you mind creating a reproduction sample?
It seems fine in my attempts

**Comment by mathieucollet** [comment]
Here is a repo that has this problem: [https://github.com/mathieucollet/storybooktest](https://github.com/mathieucollet/storybooktest)

**Comment by farnabaz** [comment]
This should fix in v4.0.1.

**Comment by niksolaz** [comment]
Hi, I've used **node 12** and **@nuxtjs/storybook: 4.0.1.**
This works for me.
I hope I have helped you

**Comment by mathieucollet** [comment]
I still run into some problems when I try to build the storybook during the IC, but it now works locally as long as I don't build it.

**Comment by mathieucollet** [comment]
@farnabaz I updated the repo with `@nuxtjs/storybook v4.0.1` to allow you reproduce the error.

With 4.0.1 I can now run `nuxt storybook` but not `nuxt storybook build`

**Comment by mathieucollet** [comment]
Damn I didn't look hard enough for the solution to my problem then :'( Sorry and thanks for the info!

**Comment by farnabaz** [comment]
Just released v4.0.2 path with the fix

**Comment by Bdthomson** [comment]
I ran into the same problem (No files matching '../.nuxt-storybook/storybook/storybook-init-framework-entry.js' were found.) and upgrading to v4 worked for me.
